mobify/platform-scaffold
# A few cleanups from during the schema work

This PR fixes some minor issues that I have found during the schema work

 **JIRA**: Accessory to https://mobify.atlassian.net/browse/WEBDATA-69
 **Linked PRs**: (links to corresponding PRs, optional)

## Changes
- Remove console.log statement from `getHighResImage`
- Rename company/apt toggle action to have distinct names on the two pages
- Use `validator` to check the email address on the checkout shipping page
- Clean up 

## How to test-drive this PR
- Preview Merlin's
- See that everything still works, including the company/apt address toggle
- See that you can use an email address on the shipping page with a `+` character in it.
